include/odp/api/spec/pool.h

```diff
@@ -200,6 +200,9 @@ typedef struct odp_pool_param_t {
 	/** Pool type */
 	int type;
 
+	/** skip buffers which cross huge page boundaries */
+	odp_bool_t skip_cross_page;
```

Comment:
It is not significant for an application, if a buffer is mapped over single or multiple physical pages. So, configuration options for this should be added into config file, rather than into the API.

Reply:
that make sense